src/components/templates/Jazzbar/JazzTab/JazzTab.tsx

```diff
@@ -41,6 +41,8 @@ import { JAZZBAR_TABLES } from "./constants";
 
 import "./JazzTab.scss";
 
+const emptyUsersList: WithId<User>[] = [];
```

Comment:
Maybe use `ALWAYS_EMPTY_ARRAY` that is frozen instead of passing a shared mutable object.

Reply:
Added in https://github.com/sparkletown/sparkle/pull/2286/commits/d663a2b389f1d204f1e1d286cc3d2a7c8d052ad0